roles/reproducer/tasks/ci_job.yml

```diff
@@ -27,34 +26,34 @@
             - bootstrap
           ansible.builtin.copy:
             src: "{{ _reproducer_basedir }}/parameters/"
-            dest: "./{{ job_id }}-params"
+            dest: "/home/zuul/{{ job_id }}-params"
```

Comment:
perhaps this instead
"{{ansible_env.HOME}}/{{ job_id }}-params"

Reply:
since the user is fixed, we can hardcode it. Also, I got bitten far too many times with delegate_to, facts and ansible internals...

Reply:
This is not the first place I see this kind of change. Can we provide more context in a comment or add it into the commit message? I'd avoid hard-coding it if we are not sure it's really really needed.

Reply:
context: delegate_to, facts and everything. Kind of a "leftover" from the time I wasn't sure about the used `user` - and I thought it would be far, far better to get absolute path. Clearer for everybody.